### ISSUE TITLE
Possible references to domain-specific selector names in applyNamespacing

### ISSUE THREAD

**Issue by jonknowles** [jotform/css.js#27]
In css.js, there appears to be logic for special behavior for selectors such as `'.form-all'`, `'#stage'`, and `'.supernova'`. Am I right in assuming that these are specific to the use case that this tool was first developed for and are not related to general CSS rules? 

The section with these selector names is here: https://github.com/jotform/css.js/blob/f9704438d2c9d9780ce2f89bd45189d19b94a686/css.js#L568-L582

**Comment by kemaldaggen** [comment]
@jonknowles Yes you are correct. Those are forgotten and should be removed since this is an open source and kind of general purpose css parser.
